### PR TITLE
Release 0.0.2

### PR DESCRIPTION
**Pull request by sfc-gh-halu**

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -9,3 +9,14 @@ Initial release supporting deploying MLflow packaged SKLearn and XGBoost models
...
 ### Changed
 ### Fixed
+
+
+## 0.0.2 (2022-02-13)
+Release to address some early PrPr feedbacks.
+### Added
+ * Support any model flavors as long as corresponding ML package is available in Snowflake conda channel.
+ * Add return for `create_deployment`.
+ * Avoid hard conda dependencies.
```
Comment (sfc-gh-sdas) [inline CHANGELOG.md:19]: expand this a bit more something like, "client does not require to run under conda env anymore".

File: CHANGELOG.md [hunk 2]
```diff
@@ -9,3 +9,14 @@ Initial release supporting deploying MLflow packaged SKLearn and XGBoost models
 ### Added
 ### Changed
 ### Fixed
+
+
+## 0.0.2 (2022-02-13)
```
Comment (sfc-gh-sdas) [inline CHANGELOG.md:14]: should we backfill the original version as well?

**Review by sfc-gh-sdas — APPROVED** [review]

File: CHANGELOG.md (hunk 2 above)
Comment (sfc-gh-halu) [reply CHANGELOG.md:14]: it's already there, hidden by the github preview lines:)

File: CHANGELOG.md (hunk 1 above)
Comment (sfc-gh-halu) [reply CHANGELOG.md]: ack

**Pull request merged**